tests/features/interactive_steps.feature

```diff
@@ -75,10 +75,82 @@ Scenario: I set text-type values
   And I tap to continue
   # Next page
   Then the question id should be "the end"
+
+@slow @i2
+Scenario: I set text-type values - check if downloading two files works
+  Given I start the interview at "all_tests"
+  And I tap to continue
+  And I set the var "double_quote_dict['double_quote_key']['dq_two']" to "true"
+  And I set the var "single_quote_dict['single_quote_key']['sq_two']" to "true"
+  And I tap to continue
+  Then the question id should be "direct standard fields"
+  # Next page
+  When I set the var "checkboxes_yesno" to "True"
+  And I set the var "checkboxes_other['checkbox_other_opt_1']" to "true"
+  And I set the var "dropdown_test" to "dropdown_opt_2"
+  And I set the var "radio_yesno" to "False"
+  And I set the var "radio_other" to "radio_other_opt_3"
+  And I set the var "text_input" to "Regular text input field value"
+  And I set the var "textarea" to "Multiline text\narea value"
+  When I set the var "direct_standard_fields" to "True"
+  # Next page
+  Then the question id should be "showifs"
+  When I set the var "show_2" to "True"
+  And I set the var "show_3" to "True"
+  And I set the var "showif_checkbox_yesno['showif_checkboxes_nota_2']" to "true"
+  And I set the var "showif_checkboxes_other['showif_checkboxes_nota_3']" to "true"
+  And I set the var "showif_yesnoradio" to "True"
+  And I set the var "showif_radio_other" to "showif_radio_multi_2"
+  And I set the var "showif_text_input" to "Show if text input value"
+  And I set the var "showif_textarea" to "Show if\nmultiline text\narea value"
+  And I set the var "showif_dropdown" to "showif_dropdown_1"
+  When I tap to continue
+  # Next page
+  Then the question id should be "buttons yesnomaybe"
+  When I set the var "buttons_yesnomaybe" to "None"
+  # Next page
+  Then the question id should be "buttons other"
+  When I set the var "buttons_other" to "button_3"
+  # Next page
+  Then the question id should be "button continue"
+  When I set the var "button_continue" to "True"
+  # Next page
+  Then the question id should be "screen features"
+  When I tap to continue
+  # Next page
+  Then the question id should be "proxy vars"
+  When I set the var "x[i].name.first" to "Proxyname1"
+  And I tap to continue
+  # Next page
+  Then the question id should be "is there another generic"
+  When I set the var "x.there_is_another" to "True"
+  # Next page
+  Then the question id should be "proxy vars"
+  When I set the var "x[i].name.first" to "Proxyname2"
+  And I tap to continue
+  # Next page
+  Then the question id should be "is there another generic"
+  When I set the var "x.there_is_another" to "False"
+  # Next page
+  Then the question id should be "first signature"
+  When I sign
+  And I tap to continue
+  # Next page
+  Then the question id should be "second signature"
+  When I sign
+  And I tap to continue
+  # Next page
+  Then the question id should be "simple doc"
+  When I download "simple-doc.pdf"
```

Comment:
I think there are supposed to be two of these lines, right?

Reply:
Oh! Is this supposed to be proof that downloading in two separate scenarios works ok?

Reply:
Note to make future issue: We might want to add something like this to our tests so that this can be confirmed as well.

Reply:
Yeah, this was to test whether downloading in two separate scenarios works okay (it does!).

I previously did have two downloads in a single scenario, but that doesn't work yet, so I took it out for now. I made an issue for fixing this later: #357